index.bs

```diff
@@ -473,11 +479,13 @@ When the user agent wants to <dfn>run animators</dfn> in a given |workletGlobalS
 
   7. For each |timeline| in the <a>animator timeline list</a> of |definition|, do the following:
 
-    - If |timeline| is "document", push a new {{AnimationTimeline}} with its "currentTime" set to
-        a <a>DOMHighResTimeStamp</a> indicating the current frame start time to the |timelines| list.
+    - If <a>Type</a>(|timeline|) is {{DocumentTimelineOptions}}, push a new {{DocumentTimeline}} 
+        created using |timeline| to the |timelines| list, with the "currentTime" set to a
```

Comment:
Given that DocumentTimelineOptions can specify an alternate originTime we should avoid saying what "currentTime" is set to, we may not need to say anything at all but if the animation timelines don't mention it we might want to say that the time source will refer to the current frame start time.